demo/api/src/app.module.ts

```diff
@@ -137,10 +132,10 @@ export class AppModule {
                 StatusModule,
                 MenusModule,
                 DependenciesModule,
-                PublicUploadModule.register({
+                FileUploadsModule.register({
                     acceptedMimeTypes: ["text/csv"],
-                    maxFileSize: config.publicUploads.maxFileSize,
-                    directory: `${config.blob.storageDirectoryPrefix}-public-uploads`,
+                    maxFileSize: config.fileUploads.maxFileSize,
+                    directory: `${config.blob.storageDirectoryPrefix}-file-uploads`,
                 }),
```

Comment:
@RainbowBunchie With this the new fileUpload module is here. Could you think of any problems with this in the brevo module?

And we could now use the comet file upload handler instead of making it custom, right?

Reply:
I just realised that I didn't test the upload. And I forgot to change the URL: https://github.com/vivid-planet/comet-brevo-module/blob/40ba7d342f242cbceff33a453fc7e5a77337a496/packages/admin/src/common/contactImport/useContactImportFromCsv.tsx#L70

I still have to do that.

Apart from that:

For the most part, PublicUpload was just renamed to FileUpload. So there shouldn't be any hard problems. 

I considered if changing the directory name would cause problems. But since all uploaded files are deleted once the import is complete, this shouldn't cause problems. Also it's a config in the project so it can be decided on a project-basis. For the demo I think it's right to rename it.

> And we could now use the comet file upload handler instead of making it custom, right?

The upload endpoint is already provided by COMET. However, the get file and delete file logic must stay like this because we don't have methods for this in COMET yet

Reply:
If the module still works the same I guess there should not be any problems here.

Reply:
I changed the URL, tested it and it works: 9fa549a18b4da47dff911097f59b294369af0a9c